contrib/storage-hive/core/src/test/java/org/apache/drill/exec/fn/hive/TestInbuiltHiveUDFs.java

```diff
@@ -43,4 +43,32 @@ public void testEncode() throws Exception {
         .baselineValues(new Object[] { null })
         .go();
   }
+
+  @Test
+  public void testReflect() throws Exception {
+    final String query = "select reflect('java.lang.Math', 'round', cast(2 as float)) as col \n" +
```

Comment:
BTW, the correct way to verify this is to check the plan produced by your limit 0 shortcut. 
But apparently, we can enhance that after yours gets in.